src/utils/displayFunctions.ts

```diff
@@ -23,10 +24,10 @@ export const makeDisplayFunctions = (brandToInfo: Map<Brand, BrandInfo>) => {
   const getDecimalPlaces = (brand: Brand) => {
     // XXX for rpc brands that don't come in the purse watcher
     if ('boardId' in brand) {
-      switch ((brand as any).boardId) {
-        case 'board0188':
-          return 6;
-      }
+      const { boardId } = brand as unknown as { boardId: string };
+      const info = wellKnownBrands[boardId];
+      assert(info, `unknown boardId ${boardId}`);
```

Comment:
crash the app in this case? I guess that's good enough for now...

Reply:
I figure, given how quickly we can fix things, better to error on noisy so we know about problems